src/index.js

```diff
@@ -51,6 +52,14 @@ export default function babel ( options ) {
 			if ( !filter( id ) ) return null;
 			if ( id === HELPERS ) return null;
 
+			const cacheKey = `${id}|${code.length}`;
```

Comment:
We have a problem with the `cacheKey` here - which is that part of the state of this plugin is the version numbers of all the plugins it is given. The `transform()` function can provide very different outputs based on the `plugins` it is given. Sadly, the plugins to not expose their version numbers.

Reply:
I'm not entirely sure about this - but babel@7 exposes new APIs like `loadPartialConfig` and others, maybe there is some new way to establish cache key nowadays? cc @loganfsmyth